src/test/java/org/broadinstitute/hellbender/tools/walkers/haplotypecaller/PartiallyDeterminedHaplotypeComputationEngineUnitTest.java

```diff
@@ -214,4 +215,42 @@ public void testDeletionUnderlapingDeterminedBases() {
         Assert.assertEquals(result.getCigar(), TextCigarCodec.decode("10M"));
         Assert.assertEquals(result.getDeterminedPosition(), DEL_AA_105.getStart());
     }
+
+    @Test
+    public void testEventsOverlapForPDHapsCode() {
```

Comment:
👍 i think we are going to appreciate having this unit test in some vague undetermined point in the future